packages/safe-upgrade/bootscript_gen.py

```diff
@@ -0,0 +1,31 @@
+script = """
```

Comment:
may add a *She-Bang* aka `#!/bin/python3` here

Reply:
@spiccinini couldn't this be done in lua? it is too simple to have it in Python ... looks unnecesary.
There is one example (very similar to yours) here: http://lua-users.org/wiki/TextTemplate
And one not that similar but much simpler here: http://lua-users.org/wiki/StringInterpolation

Reply:
@nicopace I think (please @spiccinini comment on this) this file is not going to be run on the routers, indeed it is not going to be installed on the router. So in my opinion having it in Python is not a problem

Reply:
It makes it easier for other to maintain, as it is all in one single language.
Not mandatory... But the less language diversity, the easier to get the skills to maintain or contribute

Reply:
Hi! Yes, this code is only for the develpment in case modification of the boot script is needed as it is easier to maintain in a readable form but u-boots needs it collapsed.

Yes I can port it to lua, although my confidence of the correctness will drop and as this is not central to the package is low priority for me.